### ISSUE TITLE
Select combobox and listbox variant not displaying the currently selected option on load

### ISSUE THREAD

**Issue by nielseulink** [livewire/flux#304]
The select combobox variant is not displaying the currently selected option on load, it does when you open the options list and close it.
I was hoping it would have been fixed in issue [#60](https://github.com/livewire/flux/issues/60), but ths issue still presists.

![Issue example](https://i.imgur.com/0OGzGaL.gif)

Controller:
```php
#[Title('test view')]
class Test extends Component
{
    public $creditor = 2;

    public function render()
    {
        return view('livewire.test', [
            'creditors' => [
                ['value' => 1, 'name' => 'Creditor 1'],
                ['value' => 2, 'name' => 'Creditor 2'],
                ['value' => 3, 'name' => 'Creditor 3'],
                ['value' => 4, 'name' => 'Creditor 4']
            ]
        ]);
    }
}
```

View:
```blade
<div>
    <div class="mb-4">
        <p>value: @json($creditor)</p>
        <p>options: @json($creditors)</p>
    </div>

    <flux:select variant="combobox" placeholder="Choose a creditor..." wire:model.live="creditor">
        @foreach ($creditors as $creditor)
            <flux:option wire:key="{{ $creditor['value'] }}" value="{{ $creditor['value'] }}">{{ $creditor['name'] }}</flux:option>  
        @endforeach
    </flux:select>
</div>
```

**Comment by icundin** [comment]
I have the same issue +1

**Comment by nielseulink** [comment]
It is not only a problem with the combobox variant, also with a listbox (searchable and non-searchable). A normal select works fine.

@calebporzio Any idea what this could be, or if I'm doing something wrong. Based on the thumbs up, at least 8 others have this problem.

**Comment by omniith** [comment]
I am also facing similar issues with select. The environment surrounding this issue is that if you need to livewire "update/load" value for the `wire:model.live`  in the flux:select, the values do not show in the flux:select. BUT the select is showing in the dropdown listbox/combobox.

If you load the data on mount or render, the issue is not showing. But most of the time we need to have livewire requests to update/edit fields, that is where this issue surfaces.

**Comment by calebporzio** [comment]
This should be fixed in v1.0.10 - please re-submit if I'm wrong. Thanks!
